### DISCUSSION TITLE
Different roles by cluster

### DISCUSSION THREAD

**Discussion by ClaudioMascaro** [tchiotludo/akhq#902] [Q&A] [answered]
Hey everyone, I went through the docs but I couldn't find anything about. Is that possible to control access differently between clusters? For instance I want a group to have write access for clusters "\*-sdx" and read access for clusters "\*-prd". Could be a way to filter when listing clusters (like attributes.topics-filter-regexp) or restrain write options?

**Comment by tchiotludo** [comment, marked as answer]
Hi @ClaudioMascaro : 
It's exactly the goal of this https://github.com/tchiotludo/akhq/discussions/523. 
Enhanced the ACL system to bring ACL for different cluster.

**Reply by ClaudioMascaro** [reply]
Thanks for the fast answer @tchiotludo
